manager/src/main/java/org/openremote/manager/alarm/AlarmService.java

```diff
@@ -434,24 +365,23 @@ on conflict (sentalarm_id, realm, asset_id) do nothing
     /**
      * Returns the assets linked to an alarm if the given user has access to the alarm realm.
      */
-    public List<AlarmAssetLink> getAssetLinks(Long alarmId, String userId, String realm) throws IllegalArgumentException {
-        getAlarm(alarmId, userId);
+    public List<AlarmAssetLink> getAssetLinks(SentAlarm alarm, String realm) throws IllegalArgumentException {
```

Comment:
Would stick to `Long alarmId` instead of `SentAlarm alarm`, since you don't need the full object here.